tests/Integration/Schema/Directives/CountDirectiveDBTest.php

```diff
@@ -420,4 +420,67 @@ public function testResolveRelationItemsAndCount(): void
             ],
         ]);
     }
+
+    public function testCountModelWithSingleColumn(): void
```

Comment:
I don't see how this test shows the effectiveness of `columns`. Perhaps one of the tasks should have `difficulty = null`?